src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -932,6 +932,22 @@ public uint GetNumInstanceFieldBytesIfContainsGCPointers()
         }
     }
 
+    // Subset of src\vm\typedesc.h
+    [StructLayout(LayoutKind.Sequential)]
+    internal unsafe struct TypeDesc
+    {
+        private uint m_typeAndFlags;
+        private nint ExposedClassObjectRaw;
```

Comment:
```suggestion
        private nint m_hExposedClassObject;
```

Reply:
I did this to copy the MethodDescAuxillaryData. If I do it here, I’m inclined to do the same there as well.

Reply:
It does not look good to have two different naming conventions for two fields next to each other. It would be nice to have consistent naming convention within a single type. Changing `m_typeAndFlags` to TypeAndFlags` works for me too.

Reply:
`_exposedClassObject` and `_typeAndFlags` would be the most compliant naming that follows the repo managed naming convention.

Reply:
Updating `MethodTableAuxiliaryData` right now is going to be very noisey. Will leave it for future.